packages/cosmic-swingset/test/scenario2.js

```diff
@@ -10,13 +12,33 @@ export const pspawn = (bin, { spawn, cwd }) => {
     /** @type {import('child_process').ChildProcess} */
     let child;
     const exit = new Promise((resolve, reject) => {
-      // console.debug('spawn', bin, args, { cwd: makefileDir, ...opts });
-      child = spawn(bin, args, { cwd, ...opts });
-      child.addListener('exit', code => {
+      // When stderr is otherwise ignored, spy on it for inclusion in error messages.
```

Comment:
this doesn't seem related.. what does it do?

Reply:
At some point in development, I was getting failures from packages/cosmic-swingset/test/make.test.js without any indication of the problem, because [`makeScenario2` `setup`](https://github.com/Agoric/agoric-sdk/blob/69f8e4b5e04b66df5400716251e66fbce468c86e/packages/cosmic-swingset/test/scenario2.js#L71) spawns `make` with all three standard streams set to /dev/null. Digging in uncovered the "_Include ~3 lines from child.stderr or child.stdout if present_" TODO, so I just resolved it w.r.t. stderr.

Reply:
This is super helpful, it has mystified many of us when this job fails.